### ISSUE TITLE
messages loaded number gets really really high

### ISSUE THREAD

**Issue by dominictarr** [datenknoten/ngx-ssb-client#27]
I've got it running, but it's not displaying any messages... but it seems to be loading, with the "messages in feed" counter getting really high. It starts out about 180 which seems like a reasonable number to display, but after a while it gets up to like 13000 and still hasn't displayed anything yet. I'm not sure why this but possibly a factor is that I have posted over 10 thousand ssb messages?

**Comment by enko** [comment]
@dominictarr Yes, it tries at the moment to fully retrieve your user stream. I'll remove that, as that is not needed any more.

**Comment by enko** [comment]
@dominictarr did you have the chance to try this again?
